_includes/pbc/all/upgrade-modules/upgrade-the-productconfiguration-module.md

```diff
@@ -1,4 +1,32 @@
+## Upgrading from version 0.2.* to version 1.0.*
 
+*Estimated migration time: 5 minutes*
+
+`ProductConfiguration` v1.0.0 introduces the following backward incompatible changes:
+
+* Adjusted ProductConfigurationFacade::getProductConfigurationCollection() so it is now getting product configurations by ProductConfigurationCriteria instead of ProductConfigurationFilterTransfer.
+* Impacted ProductConfigurationCartToProductConfigurationFacadeBridge::getProductConfigurationCollection() with facade changes.
```

Comment:
This bridge is not from this module. What's the reason for it to be in this migration guide?